chainstate/tx-verifier/src/transaction_verifier/signature_destination_getter.rs

```diff
@@ -77,96 +87,97 @@ impl<'a> SignatureDestinationGetter<'a> {
         tokens_view: &'a T,
         accounting_view: &'a P,
         utxos_view: &'a U,
+        spending_time: &BlockTimestamp,
```

Comment:
This is wrong. Deciding the destination has nothing to do with the timestamp. We are NOT supposed to automatically make that decision based on time. We are supposed to get the user to choose what they're signing, and then we just compare against what they decided.